internal/pkg/cli/flag.go

```diff
@@ -176,6 +177,16 @@ Cannot be specified with --%s.`, imageFlag)
 Cannot be specified with --%s.`, imageFlag)
 	storageTypeFlagDescription = fmt.Sprintf(`Type of storage to add. Must be one of:
 %s.`, strings.Join(applyAll(storageTypes, strconv.Quote), ", "))
+	storageLifecycleFlagDescription = fmt.Sprintf(`Whether the storage should be created and deleted
+at the same time as an workload or an environment.
+Must be one of: %s.`, english.OxfordWordSeries(applyAll(validLifecycleOptions, strconv.Quote), "or"))
+	storageAddIngressFromFlagDescription = fmt.Sprintf(`The workload that needs access to an
+environment storage. Must be specified with
```

Comment:
```suggestion
environment storage resource. Must be specified with
```

Reply:
<img width="551" alt="Screenshot 2023-02-17 at 3 54 18 PM" src="https://user-images.githubusercontent.com/79273084/219801112-2aed3104-fe12-4e97-90b2-f829e409c99b.png">

same here 🥲

Reply:
I still think it's a little weird, but it's a tech thing: https://helpcenter.veeam.com/docs/styleguide/tw/storage.html

Reply:
There is [some debate](https://english.stackexchange.com/questions/291079/is-storage-countable-or-uncountable-if-it-means-warehouse-or-repository) online if "storage" is a countable word, but I think it's fine in this menu context but let's add it in the docs (which you have)! Thank you!